furusiya/model/entities/npc.py

```diff
@@ -0,0 +1,23 @@
+from model.components.ai.base import AI
+from model.entities.game_object import GameObject
+
+
+class NPC(GameObject):
```

Comment:
I still resist this, and would prefer this code to go in the `Monster` class. `Stallion` won't need a complex AI.

Reply:
There is no `Monster` class. This is pretty much the same thing but under a more versatile name.